integration-testing/elasticurl_test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+set -x
+
+echo "elasticurl binary is located at $1"
+
+$1 -v TRACE example.com
+$1 -v TRACE -P -H "content-type: application/json" -i -d "{'test':'testval'}" http://httpbin.org/post
+$1 -v TRACE -o elastigirl.png https://s3.amazonaws.com/code-sharing-aws-crt/elastigirl.png
```

Comment:
comments pls

`# Sanity check, see if request/response complete without error`

I'm not really sure what the 2nd cmd is testing

`# Download a file with elasticurl and curl, ensure the results are identical`

Reply:
2nd one actually sends body data